tests/testthat/test-fit-mle.R

```diff
@@ -30,6 +30,11 @@ test_that("print() method works after optimization", {
   skip_on_cran()
   expect_output(expect_s3_class(fit_mle$print(), "CmdStanMLE"), "estimate")
   expect_output(fit_mle$print(max_rows = 1), "# showing 1 of 5 rows")
+  expect_error(
+    fit$print(variable = "unknown", max_rows = 20),
```

Comment:
I think the test error is because this needs to be `fit_mle`

Reply:
Copy-paste error. Thanks!